service/static/js/rest_api.js

```diff
@@ -141,13 +146,19 @@ $(function () {
             table += '<th class="col-md-2">ID</th>'
             table += '<th class="col-md-2">Customer ID</th>'
             table += '<th class="col-md-2">Status</th>'
+            table += '<th class="col-md-2">Product ID</th>'
             table += '<th class="col-md-2">Created On</th>'
             table += '<th class="col-md-2">Updated On</th>'
             table += '</tr></thead><tbody>'
             let firstOrder = "";
             for (let i = 0; i < res.length; i++) {
                 let order = res[i];
-                table += `<tr id="row_${i}"><td>${order.id}</td><td>${order.customer_id}</td><td>${order.status}</td><td>${order.created_on}</td><td>${order.updated_on}</td></tr>`;
+                let item_ids = []
+                for (let j = 0; j < order.items.length; j++) {
+                    let item = order.items[j];
+                    item_ids.push(item.product_id);
+                }
```

Comment:
You can use a shorter syntax for this code:
```
item_product_ids = order.items.map(item => item.product_id)
```

This is much cleaner and simple. You can also join them via a separator to show on UI, as follows:
```
item_product_ids = order.items.map(item => item.product_id).join(',')
```

Reply:
Please provide a screenshot of the updated UI